File: 2_Programação_sistemas_operacionais/2.3_Processos/Ex2.c
#include <stdio.h>
#include <stdlib.h>

int main(void)
{
	int a;
	printf("Comando: \"ls\"\n");
	a = system("ls");
	printf("Retorno: %d\n\n", a);
	
	printf("Comando: \"ls 12345.txt\"\n");
	a = system("ls 12345.txt");
	printf("Retorno: %d\n\n", a);

	printf("Comando: \"abcde\"\n");
	a = system("abcde");
	printf("Retorno de system(\"abcde\"): %d\n\n", a);
	return 0;
}

File: 2_Programação_sistemas_operacionais/2.3_Processos/Python/Ex2.py
import os

print('Comando: "ls"')
a = os.system("ls")
print("Retorno: %d" % a)

print('Comando: "ls 12345.txt"')
a = os.system("ls 12345.txt")
print("Retorno: %d" % a)

print('Comando: "abcde"')
a = os.system("abcde");
print('Retorno de system("abcde"): %d' % a)
